locations/spiders/lovinghut.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+import re
+
+from locations.items import GeojsonPointItem
+
+URL = "lovinghut.us"
+
+class LovinghutSpider(scrapy.Spider):
+    name = "lovinghut"
+    allowed_domains = [URL]
+    start_urls = (
+        'http://lovinghut.us/locations',
+    )
+
+    def normalize_time(self, time_str):
+        match = re.search(u'(.*) (am|pm|noon) \u2013 (.*) (am|pm|noon)', time_str)
+        h1, am_pm1, h2, am_pm2  = match.groups()
+        h1 = h1.split(':')
+        h2 = h2.split(':')
+
+        return '%02d:%02d-%02d:%02d' % (
+            int(h1[0]) + 12 if am_pm1 == u'pm' or am_pm1 == u'noon' else int(h1[0]),
+            int(h1[1]) if len(h1) > 1 else 0,
+            int(h2[0]) + 12 if am_pm2 == u'pm' or am_pm1 == u'noon' else int(h2[0]),
+            int(h2[1]) if len(h2) > 1 else 0,
+        )
+
+    def parse(self, response):
+        data = response.body_as_unicode()
+        stores = response.xpath('//td[@class="tg-citi"]')
+        full_addresses = response.xpath('//td[@class="tg-addr"]/text()').extract()
+        phones = response.xpath('//tr[contains(@style, "padding-bottom:10px")]/td[1]//a//text()').extract()
+        index = 0
+        for store in stores:
+            store_url = store.xpath('a/@href')[0].extract()
+
+            address_match = re.search('(.*)\,\s(.*?)$', full_addresses[index])
+            address_full, city_state = address_match.groups()
+            match = re.search('(.*?)\s(.*?)\s(.*?)$', city_state)
+            city, state, postcode = match.groups()
+
+            props = {
+                'website': store_url,
+                'addr_full': address_full,
+                'city': city,
+                'state': state,
+                'postcode': postcode,
+                'phone': phones[index],
+                'lat': '',
+                'lon': '',
+                'name': '',
+                'ref':'',
```

Comment:
You need to find a unique `ref` for each store or this spider won't output anything.